server/matcher/match.go

```diff
@@ -135,29 +137,40 @@ func (m *Matcher) Match(book Booker, queue []order.Order) (matches []*order.Matc
 			matchSet := matchLimitOrder(book, o)
 			if matchSet != nil {
 				matches = append(matches, matchSet)
-				passed = append(passed, q)
 				makers = matchSet.Makers
 			} else if o.Force == order.ImmediateTiF {
 				// There was no match and TiF is Immediate. Fail.
 				failed = append(failed, q)
 				break
 			}
 
+			// Either matched or standing unmatched => passed.
+			passed = append(passed, q)
```

Comment:
It was a bug that passed only applies to matched orders, but not newly-booked/standing unbooked orders.

Reply:
I haven't had to use it.  Will remove.

Reply:
Actually, going to remove in https://github.com/decred/dcrdex/pull/95 since it touches Matcher too.